lua/telescope/actions/set.lua

```diff
@@ -212,6 +212,43 @@ action_set.scroll_results = function(prompt_bufnr, direction)
   action_set.shift_selection(prompt_bufnr, math.floor(speed) * direction)
 end
 
+--- Select the current previewer and open it in a new scratch buffer.
+---
+---@param prompt_bufnr number: The prompt bufnr
+---@param type string: The type of selection to make
+--          Valid types include: "default", "horizontal", "vertical", "tabedit"
+action_set.select_preview = function(prompt_bufnr, type, opts)
+  opts = opts or {}
+  opts.ft = utils.get_default(opts.ft, nil)
+
+  local picker = action_state.get_current_picker(prompt_bufnr) -- picker state
+  if picker.previewer.state == nil then
```

Comment:
I inspected picker, picker.previewer and picker.previewer.state and only the latter was nil.
I will double check.

Reply:
`:Telescope find_files previewer=false`

and then

`:lua print(require "telescope.actions.state".get_current_picker(vim.api.nvim_get_current_buf()).previewer)` prints `nil` (with preview, table ref).

Reply:
I can confirm I need to check previewer.state with this function https://github.com/antoinemadec/telescope-git-browse.nvim/blob/main/lua/telescope/_extensions/git_browse.lua#L89
This when previwer is not display because the window is too small, not when previwer=false.
I added the `previewer == nil or previwer.state == nil` in https://github.com/nvim-telescope/telescope.nvim/pull/1987/commits/c5163dfbb2d210d9bd7623bf2df7199fb89f9135